code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -497,12 +503,18 @@
 		if(sheet_reagents[O.type])
 			var/obj/item/stack/stack = O
 			if(istype(stack))
+				var/list/sheet_components = sheet_reagents[stack.type]

```

Comment:
Use only `sheet_reagents[stack.type]` or `sheet_components`. Of the four subsequent uses within the scope of this variable, you use the variable for only one of them

Reply:
👍